On my 16-core machine the build fails consistently in parallel mode:

```
$ nix build -f. reaverwps
...
       last 10 log lines:
       > gcc -DCONF_DIR='"/var/db/reaver-wps-1.4/reaver"' -DREAVER_DATABASE='"/var/db/reaver-wps-1.4/reaver/reaver.db"' -Wall -g -O2 sigalrm.c -c
       > gcc -DCONF_DIR='"/var/db/reaver-wps-1.4/reaver"' -DREAVER_DATABASE='"/var/db/reaver-wps-1.4/reaver/reaver.db"' -Wall -g -O2 -Ilibwps sigint.c -c
       > gcc -DCONF_DIR='"/var/db/reaver-wps-1.4/reaver"' -DREAVER_DATABASE='"/var/db/reaver-wps-1.4/reaver/reaver.db"' -Wall -g -O2 iface.c -c
       > In file included from iface.h:40,
       >                  from iface.c:34:
       > lwe/iwlib.h:61:10: fatal error: wireless.h: No such file or directory
       >    61 | #include "wireless.h"
       >       |          ^~~~~~~~~~~~
       > compilation terminated.
       > make: *** [Makefile:67: iface.o] Error 1
```

Full build log: [reaver-wps-1.4-parallel.log.txt](https://github.com/NixOS/nixpkgs/files/7949655/reaver-wps-1.4-parallel.log.txt)

I think it will either require fixing header/binary dependencies upstream or explicitly disabling parallelism.

--- a/pkgs/tools/networking/reaver-wps/default.nix
+++ b/pkgs/tools/networking/reaver-wps/default.nix
@@ -29,6 +29,10 @@ stdenv.mkDerivation rec {
     wrapProgram $out/bin/wash   --run "[ -s ${confdir}/reaver/reaver.db ] || install -D $out/etc/reaver.db ${confdir}/reaver/reaver.db"
   '';
 
+  enableParallelBuilding = true;
+
+  patches = [ ./parallel-build.patch ];
+
   meta = with lib; {
     description = "Brute force attack against Wifi Protected Setup";
     homepage = "https://code.google.com/archive/p/reaver-wps/";

--- a/pkgs/tools/networking/reaver-wps/parallel-build.patch
+++ b/pkgs/tools/networking/reaver-wps/parallel-build.patch
@@ -1,0 +1,11 @@
+--- src/Makefile.in	2012-01-18 10:02:40.000000000 +0100
++++ src/Makefile.in	2022-01-27 19:29:19.324662149 +0100
+@@ -63,7 +63,7 @@
+ 80211.o: globule.o sql.o builder.o crc.o
+ 	$(CC) $(CFLAGS) $(INC) 80211.c -c
+ 
+-iface.o: globule.o
++iface.o: globule.o libiw
+ 	$(CC) $(CFLAGS) iface.c -c
+ 
+ sigalrm.o: globule.o

